docs/HTTP-streamingsource.md

```diff
@@ -0,0 +1,425 @@
+# HTTP Streaming Source
+
+Description
+-----------
+This plugin reads data from HTTP/HTTPS pages periodically.
+Paginated APIs are supported. For paginated APIs plugin reads available data and
+than waits for new pages to appear.
+Data in JSON, XML, CSV, TSV, TEXT and BLOB formats is supported.
+
+Properties
+----------
+
+### General
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**URL:** Url to fetch to the first page.
+The url must start with a protocol (e.g. http://).
+
+**HTTP Method:** HTTP request method.
+
+**Headers:** Headers to send with each HTTP request.
+
+**Request body:** Body to send with each HTTP request.
+
+**Track Last Page Changes:** If true, until new page appears, plugin will track the last page available and if 
```

Comment:
I don't think we need to support this in the initial version. Not sure how useful it is as it only affects things if there is no pagination, but it doesn't prevent duplicate records, only duplicate pages.

Reply:
Done removed "Track Last Page Changes" option.